code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -263,7 +263,7 @@
 	return FALSE
 
 /mob/living/carbon/Xenomorph/Ravager/process_rage_attack()
-	rage += 5 //Gain 5 rage stacks for the attack.
+	rage += 10 //Gain 10 rage stacks for the attack.
```

Comment:
move this value into either a config value, define or caste var and put the comments there.